### ISSUE TITLE
The language on the Settings doesn't match the language of the programme

### ISSUE THREAD

**Issue by vivadavid** [troyeguo/koodo-reader#839]
### I've already installed the latest developer version of Koodo Reader, this problem still exsits

- [X] Yes

### There are no similar issues on the issue list

- [X] Yes

### I've already read the document, still can't solve it.

- [X] Yes

### Operating system or browser

Windows 11

### App Version

1.5.5 (development)

### Bug Detail

Hi,

When you install Koodo and you go to _Settings_ to establish English as the language of the interface (Koodo is set to Spanish by default on my laptop following the language of my Windows install), Koodo displays _English_:

So, to sum up, the interface is in Spanish, although the settings display _English_. It's not a big deal because, to set English as the language of the interface, I just need to click on, for example, _Spanish_ and then on _English_ again, and this time the interface switches to English, but I wanted to let you know, hoping this feedback is useful.

### Screenshots (Optional)

![Koodo_language1](https://github.com/troyeguo/koodo-reader/assets/9334287/51b4c1ae-47b4-4feb-9072-12acc2d44f69)

**Comment by vivadavid** [comment]
> Thanks for the feedback, we will fix it in the next update.

Thank you!

**Comment by troyeguo** [comment]
Please update to 1.5.6

**Comment by vivadavid** [comment]
Working now, thanks!

By the way, you always provide a portable version (Koodo-Reader-1.5.6-Portable.exe), but there's also what seems to be another portable version (koodo-reader-1.5.6-x64-win.zip). I'm presuming they're the same thing presented in different forms: is that correct?

**Comment by troyeguo** [comment]
> Working now, thanks!
> 
> 
> 
> By the way, you always provide a portable version (Koodo-Reader-1.5.6-Portable.exe), but there's also what seems to be another portable version (koodo-reader-1.5.6-x64-win.zip). I'm presuming they're the same thing presented in different forms: is that correct?

Yes
